core/src/main/java/org/apache/druid/java/util/common/Bytes.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.java.util.common;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.io.Serializable;
+import java.util.Locale;
+
+@JsonSerialize(using = BytesSerializer.class)
+public class Bytes implements Serializable
```

Comment:
nit: `Bytes` seems too broad. For example, `Bytes.getValue()` doesn't seem intuitive enough to me what it would return. Maybe `HumanReadableBytes` could be a more readable name.

Reply:
I didn't come up with a nice name at first. `Bytes` is really wide used and has no specific meaning. Your suggestion is great.